services/movie.py

```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+from django.db.models import QuerySet
+from typing import Optional
+
+
+def get_movies(genres_ids: Optional[list[int]] = None,
+               actors_ids: Optional[list[int]] = None
+               ) -> QuerySet:
+    query = Movie.objects.all()
+    if genres_ids:
+        query = query.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        query = query.filter(actors__id__in=actors_ids)
+    return query
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: list[int] = None,
+                 actors_ids: list[int] = None
+                 ) -> Movie:
```

Comment:
```suggestion
def create_movie(
                 movie_title: str,
                 movie_description: str,
                 genres_ids: list[int] = None,
                 actors_ids: list[int] = None
                 ) -> Movie:
```

Reply:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]